lib/__tests__/_sendEvent.node.test.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @jest-environment node
+ */
+import AlgoliaAnalytics from "../insights";
+import { getRequesterForNode } from "../utils/getRequesterForNode";
+import { getFunctionalInterface } from "../_getFunctionalInterface";
+
+const credentials = {
+  apiKey: "testKey",
+  appId: "testId"
+};
+
+describe("_sendEvent in node env", () => {
+  let aa;
+  beforeEach(() => {
+    const instance = new AlgoliaAnalytics({ requestFn: getRequesterForNode() });
+    aa = getFunctionalInterface(instance);
+    aa("init", credentials);
+  });
+
+  it("throws when user token is not set", () => {
```

Comment:
Is `userToken: ''` a valid case? It's not tested but it works with this value.

We should track corner cases like this in tests.

Reply:
it's already handled by the API so it's not a big deal if the client lets it slip.

Reply:
What's the difference with an `undefined` `userToken` then?

Reply:
https://github.com/algolia/search-insights.js/pull/140/commits/9f8559dd866202b75c84d0c2ba69869ae78486a5